account_check_printing_report_base/i18n/es.po

```diff
@@ -103,26 +103,31 @@ msgstr "Descripción"
 #: model:ir.model.fields,field_description:account_check_printing_report_base.field_report_account_check_printing_report_base_report_check_base__display_name
 #: model:ir.model.fields,field_description:account_check_printing_report_base.field_report_account_check_printing_report_base_report_check_base_a4__display_name
 msgid "Display Name"
-msgstr "Nombre mostrado"
+msgstr "Mostrar Nombre"
 
 #. module: account_check_printing_report_base
 #: model_terms:ir.ui.view,arch_db:account_check_printing_report_base.report_check_base
 #: model_terms:ir.ui.view,arch_db:account_check_printing_report_base.report_check_base_a4
 msgid "Due Date"
-msgstr "Fecha vencimiento"
+msgstr "Fecha de Vencimiento"
```

Comment:
Upper letters must follow standard Spanish rules (no upper in the middle of a sentence except the known exceptions)

```suggestion
msgstr "Fecha de vencimiento"
```